lib/ddtrace/configuration/settings.rb

```diff
@@ -56,6 +57,18 @@ class Settings
         end
       end
 
+      settings :sampling do
+        option :default_rate do |o|
+          o.default { env_to_float(Ext::Sampling::ENV_SAMPLE_RATE, nil) }
+          o.lazy
+        end
+
+        option :rate_limit do |o|
+          o.default { env_to_float(Ext::Sampling::ENV_RATE_LIMIT, nil) }
```

Comment:
after some discussion with leads and other teams, we want the default here to be `100` like in the original spec.

However, the default sample rate of 100% (without fallback to agent controlled sampling) is still 👍 as long as this new sampler is still an opt in process (which it is).

Reply:
```suggestion
          o.default { env_to_float(Ext::Sampling::ENV_RATE_LIMIT, 100) }
```

Reply:
Done.